### PR TITLE
Gaai icons update

### PR DESCRIPTION
**Pull request by sbrin**
@gaai here's your PR update resolved

https://github.com/sbrin/lopaka/pull/92

Please check if it is good. We don't need to use gaal.ts file anymore. All images are loaded from folder specified in icon pack file

### CODE REVIEW

File: img/gaai/arrow_curved_left_down_up_5x7.png
Comment (Gaai) [inline img/gaai/arrow_curved_left_down_up_5x7.png:1]: oops. dimensions wrong way around...

File: img/gaai/door_12x16.png
Comment (Gaai) [inline img/gaai/door_12x16.png:1]: just a note: perhaps door_open...

File: img/gaai/hart_15x16.png
Comment (Gaai) [inline img/gaai/hart_15x16.png:1]: note: typo should be heart

File: img/gaai/phone_contacts_14x16.png
Comment (Gaai) [inline img/gaai/phone_contacts_14x16.png:1]: note: not symmetric.

**Comment by Gaai** [comment]
@sbrin Thanks, that's a better approach without needing the file naming list. I've discovered some naming mistakes, that I should correct. One that causes a bug, because it's got the wrong dimensions. But there's also something weird with the icon order? 
What's logic behind the current order of the icons? Anything your scripts are doing? I think an aphabetic ordering would make most sence. I've named them (categorized) them with this in mind. But somehow some icons end up in different spots. I'll see if I can figure it out myself too. 
Everything else looks good and seems to work. Probably need to do some refinements still, in terms of color inversions here and there. Some double pixels. And bring down the number of check/cross icons for example ;) Also still adding icons to my current project. So might be updating in the near future.

**Comment by Gaai** [comment]
```
const icons = computed(() => {
    return iconsList[iconsActive.value].icons.sort((a, b) => a.width * a.height - b.width * b.height);
});
```
So it seems the icons are sorted by their size. But it would be nice if they are displayed in alphabetical order.

**Comment by Gaai** [comment]
@sbrin apart from the sorting not being alphabetical but, by width. I've also noticed something weird going on with the larger 'crossed' icons. Somhow their width isn't converted well into the editor, but their names do reflect the correct width-height? 
I'll be removing the larger crosses anyway. But this could point to a possible bug in the conversion code?

**Comment by Gaai** [comment]
Added my corrections to new PR

**Comment by sbrin** [comment]
@Gaai thank you. I found the reason of wrong dimensions: it all inherits from file names. We use the part before size as image id (`crossed_11x16.png` becomes `crossed`) and we get the same image ids with the same size 😃

Will fix that soon.

**Pull request merged**